hal/include/HAL/Accelerometer.h

```diff
@@ -9,11 +9,14 @@
 
 #include "HAL/Types.h"
 
-enum HAL_AccelerometerRange : int32_t {
+/* clang-format off */
```

Comment:
Well the source code uses C++. The question is what language the headers should be written in.